bpf/bpf_lxc.c

```diff
@@ -176,7 +178,7 @@ static inline int ipv6_l3_from_lxc(struct __sk_buff *skb,
 	 * entry for destination endpoints where we can't encode the state in the
 	 * address.
 	 */
-	if ((svc = lb6_lookup_service(skb, &key)) != NULL) {
+	if (!is_defined(DATAPATH_IPVLAN) && (svc = lb6_lookup_service(skb, &key)) != NULL) {
```

Comment:
Why are we skipping load balancing?

Reply:
Ok, I've removed it. I had this originally to avoid another lookup in fast path but also because LB for local services does not work yet, but we can document this under current limitations for initial merge.